recipes/libdispatch/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = '5.10' %}
+
+package:
+  name: libdispatch
+  version: {{ version }}
+
+source:
+  url: https://github.com/apple/swift-corelibs-libdispatch/archive/refs/tags/swift-{{ version }}-RELEASE.tar.gz
+  sha256: 16e088cf12654d22658879710b9694a6fad1c94d5e5d0c597741b71fbcb3e034
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - clangxx
+    - lld  # [win]
+    # It seems libdispatch cannot be built with glibc<2.19 because of: https://bugzilla.redhat.com/show_bug.cgi?id=1009623
+    # I have no idea how Apple build Swift (libdispatch is the dep) for CentOS 7 (glibc 2.17), its CI pipline is a blackbox
+    - sysroot_linux-64 2.28  # [linux64]
```

Comment:
HELP WANTED: Either I find a way to mitigate this or I have to disable build for linux...